benchmarks/templates/benchmarks/components/left-sidebar.html

```diff
@@ -1,4 +1,18 @@
-<a class="selected" href="#">Leaderboard</a>
-<a href="#">Compare</a>
-<a href="#">Tutorials</a>
-<a href="#">Profile</a>
+{% load static %}
+<a href="http://{{ request.get_host }}/2023"><img class="brainscore-logo" src="{% static "/benchmarks/img/b-w_logo.png" %}" alt="Brain-Score Logo"></a>
```

Comment:
This looks in the screenshot and on my local checked out branch slightly off center: maybe move it a touch to the left?